plugins/generator-1.20.1/forge-1.20.1/templates/item/item.java.ftl

```diff
@@ -323,15 +323,14 @@ public class ${name}Item extends Item {
 	if (player.getAbilities().instabuild || stack != ItemStack.EMPTY) {
 		<#assign projectileClass = generator.map(projectile, "projectiles", 0)>
 		<#if projectile.startsWith("CUSTOM:")>
-			${projectileClass} projectile = ${projectileClass}.shoot(world, entity, world.getRandom());
+			${projectileClass} projectile = ${projectileClass}.shoot(world, entity, world.getRandom(), ${data.rangedAccuracy});
 		<#elseif projectile.endsWith("Arrow")>
 			${projectileClass} projectile = new ${projectileClass}(world, entity);
-			projectile.shootFromRotation(entity, entity.getXRot(), entity.getYRot(), 0, 3.15f, 1.0F);
+			projectile.shootFromRotation(entity, entity.getXRot(), entity.getYRot(), 0, 3.15f, ${data.rangedAccuracy}F);
 			world.addFreshEntity(projectile);
 			world.playSound(null, entity.getX(), entity.getY(), entity.getZ(), ForgeRegistries.SOUND_EVENTS
 				.getValue(new ResourceLocation("entity.arrow.shoot")), SoundSource.PLAYERS, 1, 1f / (world.getRandom().nextFloat() * 0.5f + 1));
 		</#if>
-

```

Comment:
This is also unrelated. Applies to both MC versions. Please use https://github.com/MCreator/MCreator/pull/4852/files tab to find such changes.

Please fix _all_ of them, not just the ones I mark